.github/workflows/ci.yml

```diff
@@ -114,94 +123,41 @@ jobs:
       - name: Run Barge
         working-directory: ${{ github.workspace }}/barge
         run: |
-          export CONTRACTS_VERSION=v2.3.0
-          bash -x start_ocean.sh --no-aquarius --no-provider --no-dashboard --with-typesense 2>&1 > start_ocean.log &
-      - run: npm ci
-      - run: npm run build
-      - run: docker image ls
-      - name: Delete default runner images
-        run: |
-          docker image rm -f node:20
-          docker image rm -f node:20-alpine
-          docker image rm -f node:18
-          docker image rm -f node:18-alpine
-          docker image rm -f debian:10
-          docker image rm -f debian:11
-          docker image rm -f ubuntu:22.04
-          docker image rm -f ubuntu:20.04
-          docker image rm -f moby/buildkit:latest
-          rm -rf /usr/share/swift/
-
-      - name: Wait for contracts deployment and C2D cluster to be ready
-        working-directory: ${{ github.workspace }}/barge
+            bash -x start_ocean.sh --with-typesense 2>&1 > start_ocean.log &
+      - name: Wait for node to be ready
         run: |
-          for i in $(seq 1 250); do
+          max_attempts=60
+          attempt=1
+          echo "Waiting for node container to be ready (timeout: 300s)..."
+          while [ $attempt -le $max_attempts ]; do
+            if [ "$(docker inspect -f '{{.State.Running}}' ocean-node-1 2>/dev/null)" = "true" ]; then
+              echo "Node container is running!"
+              break
+            fi
+            echo "Attempt $attempt/$max_attempts: Node container not ready yet, waiting..."
+            if [ $attempt -eq $max_attempts ]; then
+              echo "Error: Node container failed to start within 300 seconds"
+              docker ps -a
+              docker inspect ocean-node-1
+              exit 1
+            fi
             sleep 10
-            [ -f "$HOME/.ocean/ocean-contracts/artifacts/ready" ] && break
+            attempt=$((attempt + 1))
           done
-          cat $ADDRESS_FILE
-
+      - run: npm ci
+      - run: npm run build
+      
       - name: docker logs
         run: docker logs ocean_ocean-contracts_1 && docker logs ocean_kindcluster_1 && docker logs ocean_computetodata_1 && docker logs ocean_typesense_1
```

Comment:
These should be updated and monitor the actual containers, meaning we can safely remove `docker logs ocean_kindcluster_1, docker logs ocean_computetodata_1` and add ocean node container instead.